test/parallel/test-shadow-realm-globals.js

```diff
@@ -24,4 +24,4 @@ for (const item of items) {
   leaks.push(item);
 }
 
-assert.deepStrictEqual(leaks, []);
+assert.deepStrictEqual(leaks, ['URLPattern']);
```

Comment:
@legendecas Is this correct? How can I make sure it is not exposed?

Reply:
The interfaces exposed in `lib/internal/bootstrap/web/exposed-wildcard.js` should be annotated in the WebIDL as `[Exposed=*]`.

https://urlpattern.spec.whatwg.org/#urlpattern-class defines that `URLPattern` is annotated as `[Exposed=(Window,Worker)]`, so it should be exposed in `lib/internal/bootstrap/web/exposed-window-or-worker.js`.

Reply:
This reminds why not exposing it as `[Exposed=*]`. https://github.com/whatwg/urlpattern/pull/236 is the pending work to do it.

Reply:
Thank you. Extremely helpful as always!